src/envoy/tcp/metadata_exchange/metadata_exchange.cc

```diff
@@ -257,7 +257,7 @@ void MetadataExchangeFilter::tryReadProxyData(Buffer::Instance& data) {
     setFilterState(config_->filter_direction_ == FilterDirection::Downstream
```

Comment:
Changes in this file are covered by https://github.com/istio/proxy/pull/2565/files PR